### DISCUSSION TITLE
WIFI & USB THETHERING NOT WORKING

### DISCUSSION THREAD

**Discussion by aaravsagar** [elementary/triage#659] [Q&A]
AFTER I INSTALLED THE LATEST VERSION THE INSTALLATION AND FIRST BOOT AFTER INSTALLATION WAS GOOD AND WIFI AND USB THETHERING WORKED BUT AFTER RESTARTING THE BOTH ARE NOT WORKING

**Comment by Alcizmax** [comment]
Aaravsagar, I have the same problem. I installed Elementary 8 and did not install the system updates. Everything was working fine, but when I ran the system updates, they were downloaded correctly, and after restarting the computer, the network driver and USB ports stopped working.

**Reply by aaravsagar** [reply]
Yeah even my all storage device except the boot drive weren't recognized only my mobile was recognized and was able to access files but usb thethering didn't work

**Comment by teamcons** [comment]
From Discord:
 it was nmcli not managing my device for some reason. Solved by creating a blank file in /etc/NetworkManager/conf.d/10-globally-managed-devices.conf

terminal:
> sudo touch /etc/NetworkManager/conf.d/10-globally-managed-devices.conf

**Reply by ryonakano** [reply]
Leaving a note that this issue is fixed in https://github.com/elementary/os/pull/745. Starting from the next daily and stable ISOs this fix takes effect.

**Comment by Alcizmax** [comment]
Thanks!@teamcons@ryonakano
